ai-ml/jupyterhub/variables.tf

```diff
@@ -0,0 +1,73 @@
+variable "name" {
+  description = "Name of the VPC and EKS Cluster"
+  default     = "jupyterhub"
+  type        = string
+}
+
+variable "region" {
+  description = "region"
+  type        = string
+  default     = "us-west-2"
+}
+
+variable "eks_cluster_version" {
+  description = "EKS Cluster version"
+  default     = "1.26"
+  type        = string
+}
+
+# VPC with 2046 IPs
+variable "vpc_cidr" {
+  description = "VPC CIDR. This should be a valid private (RFC 1918) CIDR range"
+  default     = "10.1.0.0/16"
+  type        = string
+}
+
+# Routable Public subnets with NAT Gateway and Internet Gateway
+variable "public_subnets" {
+  description = "Public Subnets CIDRs. 62 IPs per Subnet/AZ"
+  default     = ["10.1.0.0/26", "10.1.0.64/26"]
```

Comment:
setting a VPC CIDR and then subnet CIDRs like this could be problematic - better to calculate based off the VPC CIDR. For example, what if someone changes just the VPC CIDR and not the rest of the values?